Documentation/Configuration/Typo3ConfVars/GFX.rst

```diff
@@ -244,6 +244,22 @@ jpg_quality
 
    Default JPEG generation quality
 
+..  index::
+    TYPO3_CONF_VARS GFX; webp_quality
+..  _typo3ConfVars_gfx_webp_quality:
+
+webp_quality
+============
+
+..  versionadded:: 13.0
+
+..  confval:: $GLOBALS['TYPO3_CONF_VARS']['GFX']['webp_quality']
+
+    :type: int
+    :Default: 85
+
+    Default WebP generation quality
```

Comment:
Do we have any information on what that means? What is the range etc?

Reply:
Like jpeg_quality. 1-100 can add it in a pre-patch also for jepg and merge it back.